.changeset/pink-dolls-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-linguist-backend': patch
+---
+
+Added support for offline language data
```

Comment:
When we change this to use the full options would be good to include an example here as well as a link to the available options. It's super handy for people who are upgrading to have this information here, saves hunting around for it

Reply:
I've updated the PR now

Reply:
Let me know if you want any other changes 😄

Reply:
Yeah, can you update this changeset to include something like this:

Added ability to use linguist-js options, a full list of the available options can be found [here](https://www.npmjs.com/package/linguist-js#API). These can be used like this:

```ts
return createRouter(
  { schedule: schedule, linguistJsOptions: { offline: true } },
  { ...env },
);
```